styles/variables-silicon.less

```diff
@@ -99,7 +99,11 @@
 @agate-input-font-size: 28px;
 @agate-input-height: 33px;
 @agate-input-opacity: 0.5;
-@agate-input-padding: 21px 32px 5px 0px;
+@agate-input-icon-display: flex;
+@agate-input-icon-align: center;
+@agate-input-padding: 21px 33px 21px 0;
+@agate-input-icon-after-padding: 21px 21px 21px 9px;
+@agate-input-icon-before-padding: 21px 9px 21px 21px;
```

Comment:
This `padding` won't work properly in RTL.
We provide the `.padding-start-end` mixins in `packages/ui/styles/mixins.less`.
```
.padding-start-end (@start, @end) {
	padding-left: @start;
	padding-right: @end;

	:global(.enact-locale-right-to-left) & {
		padding-left: @end;
		padding-right: @start;
	}
}
```

e.g> usage 1
```
	.iconBefore {
		.padding-start-end(0, @sand-inputfield-icon-spacing);
	}
```

e.g> usage 2
```
padding: @sand-popup-padding-top @sand-popup-padding-right @sand-popup-padding-bottom @sand-popup-padding-left;
.padding-start-end(@sand-popup-padding-left, @sand-popup-padding-right);
```

e.g> usage 3
```
padding-top: @sand-alert-overlay-padding-top-bottom;
padding-bottom: @sand-alert-overlay-padding-top-bottom;
.padding-start-end(@sand-alert-overlay-padding-left, @sand-alert-overlay-padding-right);
```

Reply:
To test your PR in RTL
Steps
:Input > Knobs > locale > ar-SA

Reply:
Thanks, this will be fixed